packages/terra-consumer-site/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "terra-consumer-site",
+  "version": "1.0.0",
```

Comment:
Set this version to `0.1.0` or `0.0.1`. When we do this first major release for this, lerna will auto bump the version to `1.0.0`.